lib/enigma.rb

```diff
@@ -72,5 +72,23 @@ def write_to_txt(message)
     File.write(filepath, message)
   end
 
+  def encrypt_from_file
+    filepath = user_input[1]
+    message = read_from_txt
+    encryption = encrypt(message)
+    encrypted_message = encryption[:encryption]
+    write_to_txt(encrypted_message)
+    "Created '#{filepath}' with the key #{encryption[:key]} and date #{encryption[:date]}"
+  end
 
+  def decrypt_from_file
+    filepath = user_input[1]
+    date = user_input[3]
+    key = user_input[2]
+    message = read_from_txt
+    decryption = decrypt(message, key, date)
+    decrypted_message = decryption[:decryption]
+    write_to_txt(decrypted_message)
+    "Created '#{filepath}' with the key #{decryption[:key]} and date #{decryption[:date]}"
+  end
```

Comment:
These methods are primarily just  calling previous methods in the class. Not much logic here but the important functionality is the reading and writing .txt files. the user input is the command line arguments that a user inputs when calling encrypt.rb or decrypt.rb.